src/coreComponents/physicsSolvers/solidMechanics/kernels/SolidMechanicsKernels.cmake

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
This file could arguably be named simply `config.cmake`. Or it could become `CMakeLists.txt` and be invoked via `add_subdirectory()` instead of `include()`, with the benefit of having a separate variable scope. But I don't propose doing that now, as we expect this pattern to be refactored into a reusable cmake macro/function later on.

Reply:
I agree. I had thought that this would be a cmake macro at some point. I think we want another couple of use cases before we do that.

Reply:
Oh...and we certainly need to break up `physicsSolvers` in to separate cmake files, and probably separate components.